share/src/main/java/io/zensoft/share/service/facebook/FacebookPublisherService.java

```diff
@@ -61,7 +64,7 @@ private boolean isValidImageUrl(String imageUrl) {
         try {
             image = ImageIO.read(new URL(imageUrl));
         } catch (Exception e) {
-            log.error("error checking image url", e);
+            log.info("error checking image url");
```

Comment:
What was the reason to add info instead of error?

Reply:
If image is not valid, there will be no affect to publishing. Post will be published just without image. So, in my humble opinion, it is enough to inform us

Reply:
okay, got it.